tools/benchmark_tool/openvino/__init__.py

```diff
@@ -47,6 +47,9 @@
     from openvino.runtime import save_model
     from openvino.runtime import layout_helpers
 
+    # Import backend for torch compile
+    from openvino.frontend.pytorch.torchdynamo import backend
```

Comment:
Why we have this import everywhere? benchmark, mo, ovc, pot, etc will not use torch compile, so I don't think we need this import there, do we?

Reply:
All `openvino/__init__.py` files must contain the same content. 
Otherwise, order of installation of setup.py projects can result in unexpected behavior.